xkcdpassgen.yaml

```diff
@@ -1,15 +1,16 @@
-name: xkcdpassgen
+name: xkcdpass
```

Comment:
If you name the snap the same as the command, you'll be able to just call xkcdpass from the command line.
If the app has a different name, you have to call snapname.appname.